adafruit_fxas21002c.py

```diff
@@ -170,7 +181,7 @@ def read_raw(self):
     # types.  Perhaps it doesn't understand map returns an iterable value.
     # Disable the warning.
     @property
-    def gyroscope(self):
+    def gyroscope(self) -> Tuple[float, float, float]:
```

Comment:
Looking at the return, technically the return is `List[float]`.  Unfortunately there's no way to say it's of a fixed length, so that's the best we can do for now with a type annotation.